### DISCUSSION TITLE
Sitemaps XML containing swell.store URLs

### DISCUSSION THREAD

**Discussion by Strongeyed** [swellstores/community#282] [Help] [answered]
Hi team,

Is there a way to amend the sitemap XML files to display a custom domain if hosted on Cloudflare pages?

Cheers!

**Comment by logeshswell** [comment, marked as answer]
@Strongeyed Yes, I believe you can update the custom domain in the below line:
https://github.com/swellstores/origin-theme/blob/master/modules/swell/index.js#L42

```ts
this.options.sitemap.hostname = 'example.com'
```

**Reply by Strongeyed** [reply]
@logeshswell that worked for the sitemap.xml on my private repo, thanks!
